nio/client/http_client.py

```diff
@@ -88,18 +87,13 @@
 from . import Client, ClientConfig
 from .base_client import logged_in, store_loaded
 
-if False:
-    from .crypto import OlmDevice
-    from .event_builders import ToDeviceMessage
-
 try:
     from json.decoder import JSONDecodeError
 except ImportError:
     JSONDecodeError = ValueError  # type: ignore
 
 
-logger = Logger("nio.client")
-logger_group.add_logger(logger)
+logger = logging.getLogger("nio.client")
```

Comment:
I'm personally a huge fan of this change, and thank you for taking it on!

Though I'm curious: what's your reasoning for manually-specifying the module name here instead of using:
```suggestion
logger = logging.getLogger(__name__)
```
Which also seems to be standard convention: https://docs.python.org/3/library/logging.html#logger-objects

Reply:
Oh, I basically just did an over complicated refactor here - I haven't modified the strings, I've effectively just replaced all calls of `logbook.Logger` with `logging.getLogger`

Although, I'm pretty sure (at least in most cases) `__name__` would resolve to those hardcoded strings anyway, but I might as well change it to `__name__` for consistency, I'll push in a few minutes :)